config/src/client.rs

```diff
@@ -106,3 +106,75 @@ impl Default for ConfigurationProxy {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use iroha_config_base::proxy::{Builder, LoadFromDisk};
+    use proptest::prelude::*;
+
+    use super::*;
+    use crate::torii::{uri::DEFAULT_API_URL, DEFAULT_TORII_TELEMETRY_URL};
+
+    const CONFIGURATION_PATH: &str = "../configs/client_cli/config.json";
+
+    #[allow(clippy::expect_used)]
+    fn placeholder_keypair() -> KeyPair {
+        let public_key = "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
```

Comment:
Style choice, but I'd prefer this be derived from the private key.

Reply:
This would require adding `From<PrivateKey>`, we have this [discussion](https://github.com/hyperledger/iroha/pull/2704#discussion_r970818247) in one of previous PR.
Maybe this PR is good opportunity to add this.